src/payment/strategies/paypal-commerce/paypal-commerce-credit-card-payment-strategy.ts

```diff
@@ -0,0 +1,91 @@
+import { Cart } from '../../../cart';
+import { CheckoutStore, InternalCheckoutSelectors } from '../../../checkout';
+import { OrderActionCreator, OrderRequestBody } from '../../../order';
+import { OrderFinalizationNotRequiredError } from '../../../order/errors';
+import { PaymentArgumentInvalidError } from '../../errors';
+import PaymentActionCreator from '../../payment-action-creator';
+import PaymentMethodActionCreator from '../../payment-method-action-creator';
+import { PaymentInitializeOptions, PaymentRequestOptions } from '../../payment-request-options';
+import PaymentStrategy from '../payment-strategy';
+
+import { DisableFundingType, PaypalCommerceHostedForm, PaypalCommerceInitializationData, PaypalCommerceScriptLoader, PaypalCommerceScriptOptions } from './index';
+
+export default class PaypalCommerceCreditCardPaymentStrategy implements PaymentStrategy {
+    constructor(
+        private _store: CheckoutStore,
+        private _paymentMethodActionCreator: PaymentMethodActionCreator,
+        private _paypalScriptLoader: PaypalCommerceScriptLoader,
+        private _paypalCommerceHostedForm: PaypalCommerceHostedForm,
+        private _orderActionCreator: OrderActionCreator,
+        private _paymentActionCreator: PaymentActionCreator
+    ) {}
+
+    async initialize(options: PaymentInitializeOptions): Promise<InternalCheckoutSelectors> {
+        const state = await this._store.dispatch(this._paymentMethodActionCreator.loadPaymentMethod(options.methodId));
+        const { initializationData } = state.paymentMethods.getPaymentMethodOrThrow(options.methodId);
```

Comment:
@helga1507 per discission here 
https://github.com/bigcommerce/bigcommerce/pull/36563/files#r476495898

and here 

https://github.com/bigcommerce/bigcommerce/pull/36563/files#r470550991

we need to take the `clientToken` from clientToken node, not from the `initializationData`